tests/test_rp_venv.py

```diff
@@ -76,8 +80,65 @@ def test_prepare_venv(rp_task_manager, sdist):
         'wheel']
     packages.extend(sdist_session_paths.values())
 
-    python_version = '3.8'
-
+    # We test in multiple environments, so we have to check what Python
+    # interpreter is installed in the current target resource.
+    # Note that, at this time, we use a single (user-specified) venv for the
+    # Pilot agent and for the tasks.
+    executable = os.path.join(rp_venv, 'bin', 'python3')
+    scriptlet = 'import sys; print(f"{sys.version_info.major}.{sys.version_info.minor}")'
```

Comment:
@andre-merzky is the `major.minor` version the expected form of the Python version specification, moving forward, or is it a temporary workaround? Does this seem correct for now, at least?